tests/test_postgres_plugin.rs

```diff
@@ -215,12 +220,14 @@ fn test_postgres_plugin() {
     solana_logger::setup_with_default(RUST_LOG_FILTER);
 
     unsafe {
-        let lib = Library::new("libsolana_accountsdb_plugin_postgres.so");
+        let mut filename = "libsolana_accountsdb_plugin_postgres.dylib";
```

Comment:
Initialize to "libsolana_accountsdb_plugin_postgres.so" by default

Reply:
Good catch! I assumed that I had -- updating presently.